calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/readers/IcalendarReader.java

```diff
@@ -65,6 +64,7 @@
 	public IcalendarReader()
 	{
 		super();
+		System.setProperty("net.fortuna.ical4j.timezone.cache.impl", MapTimeZoneCache.class.getName());
```

Comment:
does this need to be set back to what it was before? In other words, might this affect other parts of Sakai, or is that just not a concern?

Reply:
This property is needed in the newer version of the library, you can specify an "ical4j.properties" file or set the value at system level, I don't think this is dangerous for other parts of Sakai.

Reply:
Agree. This looks good to me.